src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const basicDiscount = 50;
+  const extraDiscount = 20;
+  const rentPrice = 40;
+
+  if (days <= 2) {
+    return days * rentPrice;
+  }
+
+  if (days >= 3 && days <= 6) {
+    return (days * rentPrice) - extraDiscount;
+  }
+
+  if (days <= 7) {
+    return (days * rentPrice) - basicDiscount;
+  }
+
+  return 0;
```

Comment:
no need for this `if` check, previous checks already would've worked by this time
```suggestion
    return (days * rentPrice) - basicDiscount;
```